### ISSUE TITLE
Do I need to apply the softmax function to my logit before calculating the CrossEntropyLoss?

### ISSUE THREAD

**Issue by h56cho** [huggingface/transformers#7827]
Hello, I am trying to compute the CrossEntropyLoss directly by using this code:
```
loss_fct = CrossEntropyLoss()
mc_loss = loss_fct(reshaped_logits, mc_labels)
```
If the reshaped_logits contain the logit values before softmax, should I apply `nn.softmax` function before I do `loss_fct(reshaped_logits, mc_labels)`? Thank you,

**Comment by sz128** [comment]
There is no need to add  the nn.softmax function. Pls. refer to the [pytorch document](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html?highlight=crossentropyloss#torch.nn.CrossEntropyLoss).
